t/03-testapi.t

```diff
@@ -291,6 +292,30 @@ subtest 'set console tty and other args' => sub {
     is($console->{args}->{foo}, 'bar');
 };
 
+subtest 'check_assert_shutdown' => sub {
+    # Test cases, when shutdown is finished before timeout is hit
+    $mod->mock(
+        read_json => sub {
+            return {ret => 1};
+        });
+    ok(check_shutdown, 'check_shutdown should return "true" if shutdown finished before timeout is hit');
+    is(assert_shutdown, undef, 'assert_shutdown should return "undef" if shutdown finished before timeout is hit');
+    $mod->mock(
+        read_json => sub {
+            return {ret => -1};
```

Comment:
why -1 specifically?

Reply:
It is because I just wanted to cover the following part in `check_shutdown` function:
```
if ($is_shutdown < 0) {
    bmwqemu::diag("Backend does not implement is_shutdown - just sleeping");
    sleep($timeout);
}
```